connectors/sources/sharepoint_online.py

```diff
@@ -698,6 +698,10 @@ async def site_pages(self, site_web_url):
             # I'm not sure if site can have no pages, but given how weird API is I put this here
             # Just to be on a safe side
             return
+        except ClientResponseError:
+            # Some site pages don't have the required columns, and it's unclear why
```

Comment:
DM me for examples of this, if you're interested. I'm moderately concerned this is a bug, but IDK.

Reply:
Should we just keep the select empty to not break the sync for pages of the site?